### ISSUE TITLE
Re-process `versions.tools-version`

### ISSUE THREAD

**Issue by daveverwer** [SwiftPackageIndex/SwiftPackageIndex-Server#801]
I just tried to pull some stats from the database about the Swift tools version that @Sherlouk kindly added support for recently, and unfortunately, we only have information for 166 packages as those are the packages that have moved their default branch forward since the change.

It would be useful, for when we roll out analysis changes like this, to be able to force an analysis pass for the three significant versions of every package (default branch, latest release, and latest beta) for when we add or change the logic in the analysis phase.

After we just chatted about this, I know this is not trivial. But, I think it's worth considering a change where we can ask a package to re-analyse, without triggering a diff in the reconciliation where packages are treated as new versions.

**Comment by finestructure** [comment]
Re-processing under way on `dev` via

```
 trap : TERM INT; while true; do ./Run re-analyze-versions --before 2020-06-25 --limit 10; sleep 10; done
```

I've started with an older date to get a smaller scope (but it affects 2077 packages nonetheless). Looking good so far, we should be done with that first test in ~40 mins.

Once that's done and validated, I'll re-run it with `--before 2021-01-13`. That's the day after we merged #877 and deployed the `targets` addition to `Product`. I.e. from `2021-01-12` we've running the latest `versions` update logic (with `tools_version` being correctly populated quite a while before `targets`).
